.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,28 @@
+name: Deploy
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
```

Comment:
We don't actually want this to run on pull requests, right? Only when pushing to master.

Reply:
oops, yes! Let me update. Thank you!